components/use-next-post.js

```diff
@@ -0,0 +1,28 @@
+import Link from 'next/link';
+
+export const useNextPost = (page, posts) => {
+  const indexNumberPath = posts.page.list;
+  const indexLength = indexNumberPath.length;
+  const indexNumber = indexNumberPath.findIndex(
+    (index) => index.content.id === page.id,
+  );
+  const nextPageData =
+    indexNumberPath[indexNumber + 1 === indexLength ? indexNumber : indexNumber + 1];
+  const nextPageTitle = nextPageData.content.title;
+  const nextPagePath = nextPageData.path.replace('blog', '');
+  const nextPageId = nextPageData.content.id;
+
+  return (
```

Comment:
Sins this is a hook it should not return jsx but an object. Yiu can add the jsx part at Page navigator

Reply:
where is the jsx part